app/views/topology/bundle.js

```diff
@@ -369,10 +369,9 @@ YUI.add('juju-view-bundle', function(Y) {
       this.db = new models.Database();
       this._cleanups.push(this.db.destroy);
     }
-    this.store = options.store;
-    if (!this.store) {
-      this.store = new juju.charmworld.APIv3({});
-      this._cleanups.push(this.store.destroy);
```

Comment:
Nope the charmstore instances do not have a destroy cycle - just remove all pointers and let the GC do it's job